analyses/cms-open-data-ttbar/reana.yaml

```diff
@@ -0,0 +1,38 @@
+inputs:
+  files:
+    - ttbar_analysis_pipeline.ipynb
+    - ttbar_analysis_pipeline.py
+    - nanoaod_inputs.json
+    - nanoad_branch_ratios.json
+    - cabinetry_config.yml
+    - cabinetry_config_ml.yml
+    - coffea.ipynb
+    - corrections.json
+    - GetIOBranches.py
+    - jetassignment_training.ipynb
+    - jetassignment_training.py
+    - make_corrections_json.py
+  directories:
+    - histograms
+    - utils
+    - models
+    - reference
+workflow:
+  type: serial
+  resources:
+    kerberos: true
+    dask:
+      image: registry.cern.ch/docker.io/alputer/agc-dask:1.0.0
+      number_of_workers: 200
+      single_worker_memory: 1Gi
+      single_worker_threads: 1
+  specification:
+    steps:
+      - name: agc
+        kubernetes_memory_limit: 5Gi
+        environment: registry.cern.ch/docker.io/alputer/agc-dask:1.0.0
+        commands:
+          - python3 ttbar_analysis_pipeline.py
+outputs:
+  files:
+    - histograms.root
```

Comment:
I think `workspace.json` would be another good addition here as a product of the workflow (it contains the statistical model).